config/locales/interface/output_elements/labels_groups/en_tables.yml

```diff
@@ -133,6 +133,115 @@ en:
         local_chp_gas: "Gas engine CHP (small-scale)"
         local_chp_wood: "Biomass CHP (small-scale)"
         local_chp_biogas: "Biogas CHP (must-run)"
+      economic_performance:
+        position: "Merit order position"
+        output_capacity: "Output capacity"
+        flh_future: "Full load hours"
+        capex: "CAPEX"
+        opex: "OPEX"
+        revenue: "Revenue"
+        profit: "Profit"
+        economic_viability: "Economic viability"
+        wtp: "WTP"
+        hour: "Hours"
```

Comment:
I'm not too sure about this unit label. If we use MWh and EUR/MWh, should we then use "h" instead?